tests/scenarios/events-query.spec.ts

```diff
@@ -247,146 +247,26 @@ export function testEventsQueryScenarios(): void {
-      expect(writeReply5.status.code).to.equal(202);
-      const write6 = await TestDataGenerator.generateRecordsWrite({ author: alice, dateCreated: firstDayOf2021, messageTimestamp: firstDayOf2021 });
-      const writeReply6 = await dwn.processMessage(alice.did, write6.message, { dataStream: write6.dataStream });
-      expect(writeReply6.status.code).to.equal(202);
-
-      fromLastDayOf2021 = await TestDataGenerator.generateEventsQuery({
-        author  : alice,
-        filters : [{ dateCreated: { from: lastDayOf2021 } }],
-        cursor  : fromLastDayOf2021Reply.cursor
-      });
-      fromLastDayOf2021Reply = await dwn.processMessage(alice.did, fromLastDayOf2021.message);
-      expect(fromLastDayOf2021Reply.status.code).to.equal(200);
-      expect(fromLastDayOf2021Reply.entries?.length).to.equal(1);
-      expect(fromLastDayOf2021Reply.entries![0]).to.equal(await Message.getCid(write5.message!));
-
-      toLastDayOf2022 = await TestDataGenerator.generateEventsQuery({
-        author  : alice,
-        filters : [{ dateCreated: { to: lastDayOf2022 } }],
-        cursor  : toLastDayOf2022Reply.cursor,
-      });
-      toLastDayOf2022Reply = await dwn.processMessage(alice.did, toLastDayOf2022.message);
-      expect(toLastDayOf2022Reply.status.code).to.equal(200);
-      expect(toLastDayOf2022Reply.entries?.length).to.equal(1);
-      expect(toLastDayOf2022Reply.entries![0]).to.equal(await Message.getCid(write6.message!));
-
-      fromLastDay2022ToLastDay2023 = await TestDataGenerator.generateEventsQuery({
-        author  : alice,
-        filters : [{ dateCreated: { from: lastDayOf2022, to: lastDayOf2023 } }],
-        cursor  : fromLastDayOf2022ToLastDay2023Reply.cursor,
-      });
-      fromLastDayOf2022ToLastDay2023Reply = await dwn.processMessage(alice.did, fromLastDay2022ToLastDay2023.message);
-      expect(fromLastDayOf2022ToLastDay2023Reply.status.code).to.equal(200);
-      expect(fromLastDayOf2022ToLastDay2023Reply.entries?.length).to.equal(1);
-      expect(fromLastDayOf2021Reply.entries![0]).to.equal(await Message.getCid(write5.message!));
-
-      // testing edge case where value equals `from` and `to`
-      fromFirstDay2022ToFirstDay2023 = await TestDataGenerator.generateEventsQuery({
-        author  : alice,
-        filters : [{ dateCreated: { from: firstDayOf2022, to: firstDayOf2023 } }],
-        cursor  : fromFirstDay2022ToFirstDay2023Reply.cursor,
-      });
-      fromFirstDay2022ToFirstDay2023Reply = await dwn.processMessage(alice.did, fromFirstDay2022ToFirstDay2023.message);
-      expect(fromFirstDay2022ToFirstDay2023Reply.status.code).to.equal(200);
-      expect(fromFirstDay2022ToFirstDay2023Reply.entries?.length).to.equal(1);
-      expect(fromLastDayOf2021Reply.entries![0]).to.equal(await Message.getCid(write5.message!));
-    });
-
     it('filters by a protocol across different message types', async () => {
       // scenario:
```

Comment:
This is the main test that validates retrieving the messages associated with a protocol

Reply:
Consider adding this info directly as in-code comment.